confgenerator/otel/modular.go

```diff
@@ -44,7 +44,7 @@ func (c Component) name(suffix string) string {
 	return c.Type
 }
 
-// configToYaml converts a tree of structss into a YAML file.
+// configToYaml converts a tree of structs into a YAML file.
```

Comment:
LOL

Reply:
My MBP keyboard introduces spurious `s` keypresses all over the place :(